contracts/FlowToken.cdc

```diff
@@ -167,32 +226,32 @@ pub contract FlowToken: FungibleToken {
         }
     }
 
-    init(adminAccount: AuthAccount) {
+    init() {
```

Comment:
I'm thinking we'll want to coordinate this change with DevEx. I'm concerned there could be a large blast radius if we make this change to the contract interface without plans for a fast follow release on dev tooling. It'd be helpful to get the scope of impact and timeline so we can communicate this planned update to the community.